airbyte-integrations/builds.md

```diff
@@ -2,121 +2,122 @@
-| Freshdesk   | [![source-freshdesk](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-freshdesk%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-freshdesk) |
-| Freshsales   | [![source-freshsales](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-freshsales%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-freshsales) |
-| Freshservice   | [![source-service](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-freshservice%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-freshservice) |
-| GitHub   | [![source-github](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-github%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-github) |
-| GitLab   | [![source-gitlab](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-gitlab%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-gitlab) |
-| Google Ads   | [![source-google-ads](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-ads%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-ads) |
-| Google Analytics v4  | [![source-google-analytics-v4](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-analytics-v4%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-analytics-v4) |
-| Google Search Console   | [![source-google-search-console](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-search-console%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-search-console) |
-| Google Sheets   | [![source-google-sheets](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-sheets%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-sheets) |
-| Google Directory API   | [![source-google-directory](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-google-directory%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-google-directory) |
+|name | status                                                                                                                                                                                                                                                                                      |
```

Comment:
There is an auto-formating issue 😉

Reply:
Hmmm is it still not good? I saw how the document looks like and looked ok to me. What exactly is wrong?

Reply:
I'm surprised by the amount of diff, I think we should just see the primetric doc added to the file. Other connectors doc should be untouched 😉

Reply:
Ok! Now it looks good! The autoformatting was changing the position of last '|' sign for some reason :smile:

Reply:
@andrzejdackiewicz , do you have a secrets/config.json ? The logs says the file specified in the `--config` file is not found: `No such file or directory: '/data/tap_config.json'`

Also, do not hesitate to rebuild the image each time you make a change to the source code.

Reply:
@Nmaxime Yes, I have the secrets/config.json file.
I am able to perform:
`python3 main.py check --config secrets/config.json`
`python3 main.py spec`
`python3 main.py read --config secrets/config.json --catalog sample_files/configured_catalog.json`
Everything is fine, i am able to read data with read.
Also everything is fine when I run on docker image:
`docker run --rm airbyte/source-primetric:dev spec`
`docker run --rm -v $(pwd)/secrets:/secrets airbyte/source-primetric:dev check --config /secrets/config.json`
`docker run --rm -v $(pwd)/secrets:/secrets -v $(pwd)/integration_tests:/integration_tests airbyte/source-primetric:dev read --config /secrets/config.json --catalog /integration_tests/configured_catalog.json`

Here is where the secret/config.json is situated:
![image](https://user-images.githubusercontent.com/88149093/188708432-89ebd8d4-e676-4504-bd32-c3b3283c343c.png)

I am building a new image every time I change anything :crying_cat_face: 
I saw that the /data/tap_config.json was not found but I could not find what it is in Airbyte docs / readme.
Pls help :cat: